Backend/templates/gitlab-ci/master-gitlab-ci.yml

```diff
@@ -134,15 +154,20 @@ BuildAndUploadDockerDev:
 
 DeployChartDev:
   extends: .deploy_chart_template
+  environment:
+    name: development
   variables:
     PROJECT: $PROJECT_DEV
```

Comment:
where is `$PROJECT_DEV`  from?

Reply:
it is in the environment  variables from the gitlab project